cmd/security-bootstrapper/res/configuration.toml

```diff
@@ -41,6 +41,8 @@ LogLevel = 'INFO'
       # this section contains the list of registry roles for EdgeX services
       # the service keys are the role names
       [StageGate.Registry.ACL.Roles]
+        [StageGate.Registry.ACL.Roles.appservice-rules-engine]
+          Description = "role for application service of rules engine"
```

Comment:
Need to also add `appservice-rules-engine` to secret store setup static list so its token is also generated

Reply:
so this is considered the default service i think because we can also add that through ENV override from secretstore-setup

Reply:
Point is if we added it statically here we need to do the same for vault token

Reply:
added `app-rules-engine` as default